pkg/secrets/pushtofile/secret_group.go

```diff
@@ -285,11 +316,7 @@ func newSecretGroup(groupName string, secretsBasePath string, annotations map[st
 
 	secretSpecs, err := NewSecretSpecs([]byte(groupSecrets))
 	if err != nil {
-		err = fmt.Errorf(
-			`cannot create secret specs from annotation "%s": %s`,
-			secretGroupPrefix+groupName,
-			err,
-		)
+		err = fmt.Errorf(`cannot create secret specs from annotation "%s": %s`, secretGroupPrefix+groupName, err)
```

Comment:
I know this isn't related to this PR specifically but while we're here, I think we should be consistent with the `unable to create...` vs `cannot create`.